client/src/PagesBody/DogFormPage/DogFormPage.js

```diff
@@ -18,13 +17,27 @@ const DogFormPage = () => {
     let modifiedInitialState = {};
     try {
       if (chosenDog !== undefined && chosenDog !== null) {
-        const dogFromDB = doggos.find((dog) => dog.dogId === chosenDog.dogId);
-        Object.keys(dogFormInitialState).forEach((key) => {
-          modifiedInitialState[key] = dogFromDB[key];
-        });
-        console.log(modifiedInitialState);
+        const requestOptions = {
+          method: 'GET',
+          redirect: 'follow',
+          credentials: 'include',
+        };
 
-        setInitialStateOfDogForm(modifiedInitialState);
+        fetch(
+          `http://localhost:27020/api/dogs/${chosenDog.dogId}`,
+          requestOptions,
+        )
+          .then((response) => response.json())
+          .then((result) => {
+            console.log(result);
+            Object.keys(dogFormInitialState).forEach((key) => {
+              modifiedInitialState[key] = result[key];
+            });
+            console.log(modifiedInitialState);
```

Comment:
potem do wyczyszczenia z logow. mysle ze powinniscie zrezygnowac z initial state jezel ijuz macie podpieta base, i reagowac tylko na to co przychodzi